docs/spec/spec.md

```diff
@@ -250,7 +252,7 @@ final mongodb:Client mongodb = check new (connection = {
 });
 ```
 
-####### Example: Create a MongoDB Client From the Connection Parameters Inline with Multiple Server Addresses
+###### Example: Create a MongoDB Client From the Connection Parameters Inline with Multiple Server Addresses
```

Comment:
Shouldn't we use sentence case here?

Reply:
In specs, we have used the title case and ina a previous meeting, we agreed not to change that for now. So I have used the title case in here as well.